CorePayments/src/main/java/com/paypal/android/corepayments/model/PatchCcoWithAppSwitchEligibilityResponse.kt

```diff
@@ -0,0 +1,58 @@
+package com.paypal.android.corepayments.model
+
+import com.paypal.android.corepayments.APIClientError
+import org.json.JSONObject
+
+data class PatchCcoWithAppSwitchEligibilityResponse(
```

Comment:
q: Can we make these data objects be `internal`? Merchants shouldn't need to create these types.

Reply:
response is used in `PaypalWebCheckoutClient` but made it restricted to Library group